ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -79,12 +79,18 @@ func (o *AviObjectGraph) BuildChildVS(key string, routeModel RouteModel, parentN
 		}
 	}
 
+	// create vhmatch from the match
+	o.BuildVHMatch(key, childNode, rule, hosts)
```

Comment:
Are these hosts taken from GW? GW can contain `*.avi.internal`. Are we adding those host as part of VH match?

Reply:
@akshayhavile the **hosts** list here is the list of hosts in httproute that have validated to be matching with the gateway listener hosts. So, these are the valid hosts that should be processed.

Reply:
I am missing something `GetGatewayRouteToHostname` function have input namespace and name . So we are providing parentNS and parentName at line number 47. I think that is for GW and GW namespace

Reply:
@akshayhavile The valid matching hosts in httproute are stored in a map with parent <gateway-ns>/<gateway-name> as key and hence also retrieved using the same.
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/ako-gateway-api/nodes/gateway_model_rel.go#L261